models/multiphase/d3q27_pf_velocity/Dynamics.c.Rt

```diff
@@ -1669,6 +1439,11 @@ CudaDeviceFunction void updateTrackers(real_t C){
 		return lower + current*(upper - lower);
 	}
 #endif
+
+
+/* Include file with the wetting boundary condition code  */
+<?RT models/multiphase/d3q27_pf_velocity/Boundary.c.Rt ?>
```

Comment:
1. it would be good for if  `rtemplate` would search for the included RT files the same way it searches for `source(...)` files. Here is the source mechanics: https://github.com/llaniewski/rtemplate/blob/master/R/RTemplate.R#L13
2. the main makefile has to discover some way that this file pulls the other file. Otherwise it will not regenerate `Dynamics.c` when `Boundary.c.Rt` is modified.

Reply:
1. Yes, I opened PR for Rtemplate
2. I adjusted the makefile, but on the way I also realised, there are more issues: 
   -  Since `Dynamics.R` can source some R files (as Travis is already doing in his own model without this PR even and  maybe others are doing as well), any change to those files could potentially alter the model and therefore it has to be rebuild. Currently only changes to `Dynamics.R` trigger such rebuilds. I changed that to be all `.R` files in the model directory instead. This is not ideal, as at will have some false positives, but I think this is better than building some kind of dependency graph of R files based on how they are sourcing each other 
   - Similarly Dynamics.c now depends on other .Rt files in the model directory